src/plots/mapbox/layers.js

```diff
@@ -193,22 +193,34 @@ function convertOpts(opts) {
             break;
     }
 
-    return { layout: layout, paint: paint };
+    return {
+        layout: layout,
+        paint: paint
+    };
 }
 
 function convertSourceOpts(opts) {
     var sourceType = opts.sourcetype;
     var source = opts.source;
-    var sourceOpts = {type: sourceType};
+    var sourceOpts = {
+        type: sourceType
+    };
     var field;
-
     if(sourceType === 'geojson') {
         field = 'data';
     } else if(sourceType === 'vector') {
         field = typeof source === 'string' ? 'url' : 'tiles';
+    } else if(sourceType === 'raster') {
+        field = 'tiles';
+        sourceOpts.tileSize = 256;
+        for(var index = 0; index < source.length; index++) {
+            var url = source[index];
+            source[index] = decodeURIComponent(url);
```

Comment:
Is this necessary? I would expect the URLs in the `mapbox.layers[i]` setting to already be decoded.